### PR TITLE
Adds gitignore, license, and install folder.

### PR DESCRIPTION
**Pull request by DevanB**

### CODE REVIEW

File: basic/.install/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  const info = await getInfo()
+
+  process.chdir('../')
+
+  replaceInFiles(['server/src/index.js'], '__PRISMA_ENDPOINT__', info.httpEndpoint)
+
+  console.log(`\
+Next steps:
+  1. Change directory: \`cd ${projectDir}/server\`
+  2. Start local server: \`yarn start\` (you can now open a Playground at http://localhost:4000)
```
Comment (marktani) [inline basic/.install/index.js]: This is not accurate - where did you copy it from? 

to open the playground we need to run `yarn dev`, which spins up a new dev server for the playground that injects a valid token into the http header

Comment (DevanB) [reply basic/.install/index.js]: I think this was copied from the react-boilerplate, prior to the move to `yarn dev`. Thanks for catching this. I'll update!

Comment (DevanB) [reply basic/.install/index.js]: Actually, this is still the same as the react-fullstack-graphql basic boilerplate, it looks like

Comment (marktani) [reply basic/.install/index.js]: did you see this: https://github.com/graphql-boilerplates/react-fullstack-graphql/pull/171/files?

Comment (marktani) [reply basic/.install/index.js]: Oh sorry, looks like that was merged in only recently 🙂

Comment (marktani) [reply basic/.install/index.js]: We decided to tackle this in another PR 🙂

**Pull request merged**